### PR TITLE
Publish dependency changes to event queue

### PR DESCRIPTION
**Pull request by ctdio**

### CODE REVIEW

File: src/UpdatesPublisher.js [hunk 1]
```diff
@@ -62,11 +83,29 @@ class UpdatesPublisher {
...
+      const {
+        name,
+        'dist-tags': distTags
+      } = npmDoc
+
+      const { latest: version } = distTags
+
+      logger.info(`Received update for package: ${name}, version: ${version}`)
+
+      const update = DependencyUpdate.wrap({
```
Comment (austinkelleher) [inline src/UpdatesPublisher.js]: I think we should still wrap and check for errors here. We shouldn't bother publishing if errors occurred and should definitely log them.

**Review by austinkelleher — CHANGES_REQUESTED** [review]

**Comment by austinkelleher** [comment]
Also we should merge your other PR merged first so that the CI can pass here.

File: package.json [hunk 2]
```diff
@@ -55,5 +55,9 @@
     "sinon": "^2.3.7",
     "uuid": "^3.1.0",
     "windbreaker-service-util": "github:windbreaker-io/windbreaker-service-util"
+  },
+
+  "ava": {
+    "require": [ "require-self-ref" ]
```
Comment (ctdio) [inline package.json:61]: No need to add `require('require-self-ref')` to the top of every test file with this.

**Comment by ctdio** [comment]
@Druotic @austinkelleher pls review.

**Review by Druotic — APPROVED** [review]

**Pull request merged**